src/sql/parts/jobManagement/views/jobHistory.component.ts

```diff
@@ -44,6 +47,9 @@ export class JobHistoryComponent extends Disposable implements OnInit, OnDestroy
 
 	private prevJobId: string = undefined;
 	private isVisible: boolean = false;
+	private stepRows: StepsViewRow[] = [];
+	private _showSteps: boolean = false;
```

Comment:
there is inconsistent usage of `_` prefix on private variable.  Can you please make these usage consistent in this file?